vignettes/how-to-update-released-site.Rmd

```diff
@@ -0,0 +1,272 @@
+---
+title: "How to update a released site"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{How to update a released site}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+## Checklist
+
+If you're already up-to-speed, here's the checklist:
+
+* Update to usethis >= v2.1.5.9000, maybe with
```

Comment:
I think you could collapse these first four bullet points to — make sure the development version of the site looks the way that you want it before embark on updating the released site.